### PR TITLE
Issue28 newqueue toqueue

### PR DESCRIPTION
**Pull request by adinabruce**
created arrows that points to the corresponding self. front and self. rear.
arrows only work for the insert at rear and delete from front functions, that are used in a queue, however the buttons that are used for a deque are still functional, just without arrows.
dealt with the issue of wraparound.
fixed bugs left over from converting the previous queue function to use the visualization app.

### CODE REVIEW

File: PythonVisualizations/newQueue.py [hunk 1]
```diff
@@ -0,0 +1,473 @@
...
+ARRAY_X0 = 100
+ARRAY_Y0 = 100
+
+<<<<<<< Updated upstream:PythonVisualizations/newQueue.py
+class Queue(object):
+    Element = recordclass('Element', ['val', 'color', 'display_shape', 'display_val'])
+    Element.__new__.__defaults__ = (None,) * len(Element._fields)
+=======
+class Queue(VisualizationApp):
+>>>>>>> Stashed changes:PythonVisualizations/SimpleQueue.py
```
Comment (JMCanning78) [inline PythonVisualizations/newQueue.py:18]: This is an unresolved merge conflict.

File: PythonVisualizations/newQueue.py [hunk 2]
```diff
@@ -0,0 +1,473 @@
...
+            
+            # update window
+            window.update()
+            
+        
+=======
+            self.list[self.rear] = drawable(val, Queue.colors[Queue.nextColor], cell, cell_val)
+            didInsert = True
+
+>>>>>>> Stashed changes:PythonVisualizations/SimpleQueue.py
```
Comment (JMCanning78) [inline PythonVisualizations/newQueue.py:111]: This is an unresolved merge conflict.

Comment (adinabruce) [reply PythonVisualizations/newQueue.py:111]: have resubmitted pull request with #94 does it work now?

Comment (JMCanning78) [reply PythonVisualizations/newQueue.py:111]: Yes, PR #94 fixed the merge conflicts and the changes to files that did not need changes.  I have reviewed PR #94 and we can switch to that one for the follow-up.  I will close this PR.

**Comment by JMCanning78** [comment]
Closing this now since all follow-ups are addressed by PR #94.

**Pull request closed without merging**